content/docs/rfcs/17/README.md

```diff
@@ -161,10 +168,11 @@ message RateLimitProof {
 
 message WakuMessage {
   bytes payload = 1;
-  string contentTopic = 2;
-  uint32 version = 3;
-  double timestamp = 4;
-+ RateLimitProof rate_limit_proof = 21;
+  string content_topic = 2;
+  optional uint32 version = 3;
+  optional sint64 timestamp = 10;
+  optional bool ephemeral = 31;
+  bytes rate_limit_proof = 21;
```

Comment:
Perhaps it makes sense for `rate_limit_proof` to be explicitly `optional`? This would allow for most decoders to provide an `Option.isNone()` check for the proof (e.g. on content topics not RLN-protected).

Reply:
Addressed in d46aa3b